### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Eclipse Azure Sign In_Screen Reader: Narrator is not reading the error messages that comes when user does not provide any Authentication file 

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3311]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #Narrator

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen reader: Narrator
MAS Violated: MAS1.3.1

**Note:** Similar issue is found for all the Error messages in entire application
1.	Select Subscriptions (When no subscription is selected)
2.	Create Authentication Files (When no subscription is selected)
3.	New Dynamic Web App
2.	Create New Web App
3.	New JSP File

**Repro Steps:** 
1) Start Narrator and Open Java EE - Eclipse application 
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Tab till Service principle and select the radio button
5) without providing any ‘Authentication file’ tab till sign in button and press enter
6) Verify if Narrator is reading the error message that comes when user does not provide any Authentication file

**Actual Result:** 
Narrator is not reading the error messages that comes when user does not provide any Authentication file

**Expected Result:** 
Narrator should read the error messages that comes on the screen when user does not provide any file

**User Impact:** 
Screen reader users will not know the error that occurred if screen reader does not read the error message to the users

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_1_error message is not read by narrator.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451024/Bug_1_error.message.is.not.read.by.narrator.pptx)

![Bug_1_New Dynamic file](https://user-images.githubusercontent.com/41109135/62202565-24cf8180-b3a7-11e9-94b2-758501bed0ee.png)
![Bug_1_New Dynamic web app](https://user-images.githubusercontent.com/41109135/62202567-24cf8180-b3a7-11e9-98aa-6117be6816e2.png)
![Bug_1_Select Subcriptions](https://user-images.githubusercontent.com/41109135/62202569-24cf8180-b3a7-11e9-8e42-ef12d441556c.png)
![Bug_1_Create Authentication Files](https://user-images.githubusercontent.com/41109135/62202563-239e5480-b3a7-11e9-90f8-82b036e9cfd8.png)
![Bug_1_Select a JSP file](https://user-images.githubusercontent.com/41109135/62208028-52bac300-b3b3-11e9-864e-c397cf2631da.png)
![Bug_1_Create a web app](https://user-images.githubusercontent.com/41109135/62208029-52bac300-b3b3-11e9-87de-f2052f4bcd27.png)

**Comment by Flanker32** [comment]
Narrator could read the error message in `TitleAreaDialog`, you may click the line you want to read and press `CapsLock` + 'I'.

**Comment by Flanker32** [comment]
- Importance of experience: Medium, Users who use screen reader will not know the error that occurred if screen reader does not read the error message to the users.
- Quality of experience: not impacted. Narrator could read the error message by click the line and press `CapsLock` + `I`
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
For app service creation errors, we have copied the error message to title text box which could be read by narrator, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_PR/3749/Azure/processDownloadRequest/artifacts/WindowsAzurePlugin4EJ.PR.3749.08-30-2019.zip)

**Comment by Flanker32** [comment]
@Tanushagottipati Please have a review,

**Comment by Flanker32** [comment]
Linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @Tanushagottipati please help us review it, thanks!

**Comment by Tanushagottipati** [comment]
This issue is not observed with NVDA
![Screenshot (422)](https://user-images.githubusercontent.com/41109135/64427997-3ad01080-d0d0-11e9-9a07-760d3081de28.png)

**Comment by Tanushagottipati** [comment]
External Bug: External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=550843

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by soumyatippapuram** [comment]
This issue is also observed in the latest test pass for the below screens: 
**Azure Sign-in page**
![image](https://user-images.githubusercontent.com/41108985/78907724-b1861f80-7a9e-11ea-9eab-44a8fe31501c.png)

**Azure Sign-in -> Device login ->Your Subscriptions**
![image](https://user-images.githubusercontent.com/41108985/78907852-e1352780-7a9e-11ea-92b5-c8799c080f5c.png)


**Azure Sign-in -> Service Principle ->Create Authentication files**
![image](https://user-images.githubusercontent.com/41108985/78907802-cf538480-7a9e-11ea-96b7-fd40c2bd9fd1.png)


**Create a Scala Project**
![image](https://user-images.githubusercontent.com/41108985/78907891-f1e59d80-7a9e-11ea-82fc-2cc02846040a.png)


**Create new file**
![image](https://user-images.githubusercontent.com/41108985/78907928-fd38c900-7a9e-11ea-8732-7f01be0e13fc.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749663
